### PR TITLE
Suggests improvement to introduction

### PR DESCRIPTION
**Pull request by clausmullie**

### CODE REVIEW

**Review by ericherman — APPROVED** [review]

**Comment by ElenaFdR** [comment]
What was the logic behind changing the intro to the bulleted list of other things you might be interested in? It's very cognitively dissonant to include a list of stuff not on this page; presumably every webpage could have an infinite version of that list at the top.

File: index.md [hunk 1]
```diff
@@ -5,9 +5,12 @@ redirect_from:
...
 # Resources and projects
 
-Built when we discovered a need in our own [codebase stewardship practice](https://publiccode.net/codebase-stewardship/), these resources are all openly licensed. We'd love to hear how you use these.
+Below you can find a set of resources we have developed to support our [codebase stewardship practice](https://publiccode.net/codebase-stewardship/).
+These resources are all openly licensed, so please feel free to reuse them.
+We'd love to hear from you if you do.
+We are also always open for contributions and ideas for improvements.
 
-See more of our work in:
+Note this page does not include:
```
Comment (Ainali) [inline index.md:13]: Is there some punctuation or word missing in this sentence?

**Comment by clausmullie** [comment]
I found it a bit confusing that the first list on the resources page is a `See more of our work in:` with a list of links to other places. 

But yes, the new wording isn't optimal either. If f we want the header to orient readers to other things that are not resources, how about

```
in addition to the resources below, also check out:
*
*
*
```

Or if we keep the current wording of `see more of our work`, move it to the bottom of the page?

File: index.md (hunk 1 above)
Comment (clausmullie) [reply index.md:13]: Not that I am aware of - although it is perhaps more spoken English than written English.

@ElenaFdR? :angel:

Comment (Ainali) [reply index.md:13]: I am thinking that either "Note, this page does not include:", "Note: this page does not include:" or "Note! This page does not include:" would be possible punctuations. Alternatively, "Note that this page does not include:" would sound better in my ears.

But thinking about it, it also seems odd as it is a very incomplete list of what this page does not include. If we have this bullet point list, I think going with your suggested "in addition to the resources below, also check out:" would be best.

**Pull request merged**